internal/app/shared/remediation.go

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package shared
+
+import (
+	"time"
+
+	"github.com/cloudoperators/heureka/internal/entity"
+)
+
+// RemediationTimelineConfig holds the configuration for remediation timelines
+type RemediationTimelineConfig struct {
+	LowDays      int
+	MediumDays   int
+	HighDays     int
+	CriticalDays int
+	DefaultDays  int
+}
+
+// DefaultRemediationConfig provides default values for remediation timelines
+var DefaultRemediationConfig = RemediationTimelineConfig{
+	LowDays:      180, // 6 months
+	MediumDays:   90,  // 3 months
+	HighDays:     20,
+	CriticalDays: 7,
+	DefaultDays:  365, // 1 year
+}
```

Comment:
I like it, maybe we can make in addition a comment to make them configurable via env variables in future